src/data/events/events.js

```diff
@@ -53,7 +53,8 @@ const events = [
 		date: getDateTime(2022, 10, 19, 18),
 		location: 'Haines Hall A2',
 		imgFilePath: 'event/2022f-hackschool-banner.png',
-		detailLink: 'https://discord.gg/rup2p6fxA5'
+		detailLink: 'https://discord.gg/rup2p6fxA5',
```

Comment:
Should we be changing the detailLink to the facebook event? Also, if this is the same link as last week's, then feel free to update all detailLink's to the facebook event.